src/components/DateModal.tsx

```diff
@@ -0,0 +1,141 @@
+import { useState, forwardRef } from "react";
+import { Button, Icon } from "design-react-kit";
+import { Modal, ModalHeader, ModalBody, ModalFooter } from "reactstrap";
+import { format } from "date-fns";
+import DatePicker from "react-datepicker";
+
+const DateModal = ({
+  from: propDateFrom,
+  to: propDateTo,
+  onSubmit,
+  label,
+  title
+}: {
+  from?: Date;
+  to?: Date;
+  onSubmit(propDateFrom: Date | undefined, propDateTo: Date | undefined): void;
+  label: string;
+  title: string;
+}) => {
+  const [dateFrom, setDateFrom] = useState<Date | undefined>(propDateFrom);
+  const [dateTo, setDateTo] = useState<Date | undefined>(propDateTo);
+  const [isOpenDateModal, setOpenDateModal] = useState(false);
```

Comment:
Prefer to call as destructured into value + setter pair 

```suggestion
  const [showOpenDateModal, setShowOpenDateModal] = useState(false);
```

Reply:
fixed here https://github.com/pagopa/cgn-onboarding-portal-frontend/pull/324/commits/adf5e960e66781a12faf994d3eaa77b5e7db0df4